SlevomatCodingStandard/Helpers/TokenHelper.php

```diff
@@ -123,6 +127,41 @@ public static function findNext(File $phpcsFile, $types, int $startPointer, ?int
 	 * @return int[]
 	 */
 	public static function findNextAll(File $phpcsFile, $types, int $startPointer, ?int $endPointer = null): array
+	{
+		return self::findNextAllBatch($phpcsFile, $types, $startPointer, $endPointer);
```

Comment:
I don't like the two new `public` methods. And I don't like the `Positions` class either. I would be happy with everything in `findNextAll()` method.

Reply:
Will make two new methods private. Are you sure we want to inline Positions logic, it will be much harder to test and slice logic is not trivial, but i can inline it if you wish.

Reply:
You can create more private methods but I don't like any new `public` classes when not neccessary.